code/game/objects/items/devices/PDA/PDA_types.dm

```diff
@@ -6,13 +6,26 @@
 	icon_state = "pda-clown"
 	desc = "A portable microcomputer by Thinktronic Systems, LTD. The surface is coated with polytetrafluoroethylene and banana drippings."
 	ttone = "honk"
+	var/slipvictims = list() //Haven - Ports crew objectives from cit. Credit to Deathride58
 
 /obj/item/device/pda/clown/Initialize()
 	. = ..()
-	AddComponent(/datum/component/slippery, 120, NO_SLIP_WHEN_WALKING, CALLBACK(src, .proc/AfterSlip))
 
-/obj/item/device/pda/clown/proc/AfterSlip(mob/living/carbon/human/M)
-	if (istype(M) && (M.real_name != owner))
+	AddComponent(/datum/component/slippery, 120, NO_SLIP_WHEN_WALKING)
+
+/obj/item/device/pda/clown/ComponentActivated(datum/component/C)
```

Comment:
After reverting above change, revert this proc back to AfterSlip

Reply:
Rephrase? Not quite parsing.